app/components/works/detail_component.rb

```diff
@@ -44,6 +44,14 @@ def depositor
       "#{work.depositor.sunetid} (#{work.depositor.name})"
     end
 
+    def owner
+      "#{work.owner.sunetid} (#{work.owner.name})"
+    end
+
+    def owner?
```

Comment:
Can this be `show_owner?` or `owner_different_from_depositor?`?

Reply:
Yup. Switched to `show_owner?`